assets/js/components/LoadpointSettingsModal.vue

```diff
@@ -215,22 +215,22 @@ export default {
 	},
 	computed: {
 		phasesOptions() {
-			if (this.chargerPhysicalPhases == 1) {
-				// known fixed phase configuration, no settings required
-				return [];
-			}
-			if (this.chargerPhases1p3p) {
+			if (this.chargerPhaseSwitching) {
 				// automatic switching
 				return [PHASES_AUTO, PHASES_3, PHASES_1];
 			}
+			if (this.chargerPhaseReading) {
+				// known fixed phase configuration, no settings required
+				return [];
+			}
```

Comment:
Wir haben hier einen Fall übersehen der erklärt wieso die Logik so ist wie sie ist und auch erklärt wieso wir die Anzahl der gelesenen Phases brauchen:

`chargerPhysicalPhases` gibt die Anzahl der aktiven Phasen an, die in die Wallbox reingehen. Das ist unabhängig von Wallbox-internen Einstellung (1p3p). Heißt bei meiner Easee bekomme ich immer `chargerPhysicalPhases: 3`, weil der Elektriker sie mit 3p angeklemmt hat.

Wäre diese 3-phasentaugliche Wallbox nur mit einer Phase angeklemmt `chargerPhysicalPhases == 1` brauchen wir den Phasenauswahldialog (auto, 3p, 1p) nicht anzeigen.

Ich hab gerade noch keine gute Idee, wie wir das so umstellen können, dass es einfacher verständlich ist.

Reply:
Auf jeden Fall wird es dann falsch initialisiert. Denn mit dem PhaseGetter hat das dann nichts zu tun. Ich würde sogar sagen, dass man den Fall ignorieren kann denn er wäre durch Einstellen auf 1p ebenso abgedeckt, zumal das Backend ihn ohnehin ignoriert.

Reply:
> er wäre durch Einstellen auf 1p ebenso abgedeckt

Auf der reinen Technik/Regelungsebene ist das so. Aus Nutzersicht ist das aber unerwartet/verwirrend, wenn wir dem Nutzer Auto,1P,3P anbieten, diese Einstellung aber keinen Effekt hat.

Aber ja, vmtl. ist das eher ein Edge-Case. Den werden wir aber eingebaut haben weil es einen Bedarf dafür gab.

Wir können das aber gerne in diese neue (einfacher zu verstehende) Richtung umbauen. Wenns da Feedback oder Irritation gibt können wir den Weg ja noch mal überdenken bzw. überlegen wie man das eleganter lösen kann.